source/features/toggle-files-button.css

```
@@ -12,7 +12,8 @@
 	transform: rotate(0.5turn);
 }
 
-.rgh-files-hidden .files {
+.rgh-files-hidden .files,
```

Mark the old style with this code so in the future we know what to drop
```suggestion
.rgh-files-hidden .files, /* Pre "Repository refresh" layout */
```